lib/taiko.js

```diff
@@ -3121,6 +3122,16 @@ getPlugins().forEach(pluginName => {
   }
 });
 
+/**
+ * Lets you read the global configurations.
+ *
+ * @example
+ * getConfig("retryInterval");
+ *
+ * @param {String} paramName - Specifies the name of the paramter you want to get (optional).  If not specified, returns a shallow copy of the full global configuration. 
+*/
+module.exports.getConfig = getConfig;
```

Comment:
Unit test missing.

Reply:
@saikrishna321 Working on it, but is there a reason that in the `setConfig` test, the `allowedConfig` string is manually created instead of made from the keys of `defaultConfig`?  And should I do the same with the `getConfig` tests?

Reply:
@moogleberry 
> allowedConfig string is manually created instead of made from the keys of defaultConfig

I don't see a reason why not use `defaultConfig`.  Definitely a place to improve the test. It would be great if you can update the test pick from `defaultConfig`.

Reply:
@saikrishna321 Ok did that